src/UnitTests/UTHelper/unit_test_helper.h

```diff
@@ -17,10 +17,10 @@
 #ifndef UNIT_TEST_HELPER
 #define UNIT_TEST_HELPER
 
-#ifdef WIN32
-#define VLD_FORCE_ENABLE 1
-#include <vld.h>
-#endif
+//#ifdef WIN32
+//#define VLD_FORCE_ENABLE 1
+//#include <vld.h>
+//#endif
```

Comment:
Uncomment

Reply:
Uncommented and added check to only include VLD on Debug builds (VLD report does not provide useful information on Release builds)